pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/dashboard/BarChartStrategy.java

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.dsm.model.dashboard;
+
```

Comment:
what is this class intended to do?  please add comment

Reply:
about the "builders vs lombok" question we should make a DDP team decision!